packs/validator-0.0.40/values.yaml

```diff
@@ -0,0 +1,596 @@
+        containerSecurityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+        image:
+          repository: quay.io/validator-labs/validator
+          tag: v0.0.40  # x-release-please-version
+        resources:
+          limits:
+            cpu: 500m
+            memory: 512Mi
+          requests:
+            cpu: 10m
+            memory: 64Mi
+        sinkWebhookTimeout: 30s
+      replicas: 1
+      serviceAccount:
+        annotations: {}
+    kubernetesClusterDomain: cluster.local
+    metricsService:
+      ports:
+      - name: https
+        port: 8443
+        protocol: TCP
+        targetPort: https
+      type: ClusterIP
+    
+    # Optional environment variable configuration
+    env: []
+    # - name: HTTP_PROXY
+    #   value: http://192.168.1.100:8080
+    # - name: HTTPS_PROXY
+    #   value: http://192.168.1.100:8080
+    # - name: NO_PROXY
+    #   value: foo.bar.com,127.0.0.1
+    
+    # Optional proxy configuration. If enabled, the secret containing your proxy CA certificate
+    # will be mounted in the manager container and configured via an init container.
+    proxy:
+      enabled: false
+      # The image used by the init container. Must include the update-ca-certificates command.
+      image: quay.io/validator-labs/validator-certs-init:1.0.0
+      # The name of a secret containing a proxy CA certificate.
+      secretName: proxy-cert
+      # If false (using an existing secret), the key whose value is the CA certificate must end with '.crt'.
+      createSecret: false
+      # Raw CA certificate, required if createSecret is true
+      caCert: |
+        -----BEGIN CERTIFICATE-----
+        <your certificate content here>
+        -----END CERTIFICATE-----
+    
+    # Optional sink configuration
+    sink: {}
+      # type: alertmanager
+      # secretName: alertmanager-sink-secret
+      # endpoint: "https://alertmanager.alertmanager.svc.cluster.local:9093"
+      # insecureSkipVerify: "false"
+      # caCert: ""    # (TLS CA certificate, optional)
+      # username: ""  # (HTTP basic auth, optional)
+      # password: ""  # (HTTP basic auth, optional)
+    
+      # OR
+      # type: slack
+      # secretName: slack-sink-secret
+      # apiToken: ""
+      # channelId: ""
+    
+      # By default, a secret will be created. Leave the above fields blank and specify 'createSecret: false' to use an existing secret.
+      # WARNING: the existing secret must match the format used in sink-secret.yaml
+      # createSecret: true
+    
+    cleanup:
+      image: gcr.io/spectro-images-public/release/spectro-cleanup:1.2.0
+      grpcServerEnabled: true
+      hostname: validator-cleanup-service
+      port: 3006
+    
+    # Validation plugin charts
+    plugins:
+    - chart:
+        name: validator-plugin-aws
+        repository: "https://validator-labs.github.io/validator-plugin-aws"
+        version: v0.0.26
+      values: |-
```

Comment:
Some of these configs are also in the validator chart. Do we need the same set of options in the sub chart as well? Can we not use the config from the parent chart?